### PR TITLE
Adding Sprocket factor 

### PR DESCRIPTION
**Pull request by romainbiard**
Adding Sprocket factor to configure the speed of translation of the navbar regarding the scrollview (factor of 2 means that a move of 2 pts in scrollview move the navbar from 1px).

Thank you !
Romain

### CODE REVIEW

File: Source/ScrollingNavigationController.swift [hunk 1]
```diff
@@ -87,7 +88,7 @@ open class ScrollingNavigationController: UINavigationController, UIGestureRecog
    - parameter scrollableView: The view with the scrolling content that will be observed
    - parameter delay: The delay expressed in points that determines the scrolling resistance. Defaults to `0`
    */
-  open func followScrollView(_ scrollableView: UIView, delay: Double = 0) {
+  open func followScrollView(_ scrollableView: UIView, delay: Double = 0, sprocketFactor: Double = 1) {
```
Comment (andreamazz) [inline Source/ScrollingNavigationController.swift]: Please add the new parameter to the documentation comment.

**Comment by andreamazz** [comment]
Hi @romainbiard 
This looks good, thank you.  
I'm not a big fan of the name of the parameter though :)  
I think it could be made more clear, something like "scrollSpeed" or "scrollRatio"  
What do you think?

**Comment by andreamazz** [comment]
Beautiful. Thanks 👍

**Comment by romainbiard** [comment]
Hi @andreamazz and thank you for your feedback.
I've renamed it to scroolSpeedFactor and added some docs missing from previous commit (Sorry for that).
Great work BTW :)

**Comment by romainbiard** [comment]
When do you plan to release a new version on Cocoapod ?

**Comment by andreamazz** [comment]
Just finished pushing `3.1.0`.  

Cheers

**Comment by romainbiard** [comment]
Excellent ! 👏

**Pull request merged**